smdebug/tensorflow/keras.py

```diff
@@ -5,6 +5,7 @@
 import tensorflow.compat.v1 as tf
 from tensorflow.python.distribute import values
 from tensorflow.python.framework.indexed_slices import IndexedSlices
+from tensorflow.python.util import nest
```

Comment:
is this valid in both TF1 and TF2?

Reply:
This import is valid in both TF1 and TF2